pkg/trace/config/apply.go

```diff
@@ -348,7 +348,14 @@ func (c *AgentConfig) loadDeprecatedValues() error {
 		c.BucketInterval = d * time.Second
 	}
 	if cfg.IsSet("apm_config.receiver_timeout") {
-		c.ReceiverTimeout = cfg.GetInt("apm_config.receiver_timeout")
+		c.ReceiverReadTimeout = cfg.GetInt("apm_config.receiver_timeout")
+		c.ReceiverWriteTimeout = cfg.GetInt("apm_config.receiver_timeout")
+	}
+	if cfg.IsSet("apm_config.receiver_read_timeout") {
```

Comment:
You'll need to add some extra lines around here: https://github.com/DataDog/datadog-agent/blob/05e067be4fac699854971b6ca8d58cc724fd30ee/pkg/config/apm.go#L57

Something like:
```go
	config.BindEnv("apm_config.receiver_write_timeout", "DD_APM_RECEIVER_WRITE_TIMEOUT")                             //nolint:errcheck
	config.BindEnv("apm_config.receiver_read_timeout", "DD_APM_RECEIVER_READ_TIMEOUT")                             //nolint:errcheck

```